processing/src/test/java/org/apache/druid/query/context/ResponseContextTest.java

```diff
@@ -285,6 +285,8 @@ public void deserializeTest() throws IOException
     );
     Assert.assertEquals("string-value", ctx.get(ResponseContext.Key.ETAG));
     Assert.assertEquals(100, ctx.get(ResponseContext.Key.NUM_SCANNED_ROWS));
+    ctx.add(ResponseContext.Key.NUM_SCANNED_ROWS, 10L);
+    Assert.assertEquals(110L, ctx.get(ResponseContext.Key.NUM_SCANNED_ROWS));
```

Comment:
Is it possible to add a similar assert for `CPU_CONSUMED_NANOS`?

Reply:
the code is equivalent making a test sort of redundant, but yes anything is possible.

Reply:
It's still useful as a regression test, IMO.